src/Microsoft.Security.Utilities.Core/LowConfidenceClassifiers/SEC101_060_LooseSasSecret.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Web;
+
+namespace Microsoft.Security.Utilities
+{
+    public class LooseSasSecret : RegexPattern
+    {
+        public LooseSasSecret()
+        {
+            Id = "SEC101/060";
```

Comment:
The Release Notes sets the rule ID to `SEC101/051`. This is a bug. What's the correct rule ID?

Reply:
`060` thanks for noticing.